apps/storybook/src/AxialSelectToZoom.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { AxialSelectToZoomProps } from '@h5web/lib';
+import { mockValues, useDomain } from '@h5web/lib';
+import { Zoom } from '@h5web/lib';
+import {
+  assertDefined,
+  DataCurve,
+  getAxisValues,
+  AxialSelectToZoom,
+  Pan,
+  ResetZoomButton,
+  VisCanvas,
+} from '@h5web/lib';
+import type { Meta, Story } from '@storybook/react';
+
+import FillHeight from './decorators/FillHeight';
+
+const Template: Story<AxialSelectToZoomProps> = (args) => {
+  const domain = useDomain(mockValues.oneD);
+  assertDefined(domain);
+
+  return (
+    <VisCanvas
+      abscissaConfig={{
+        visDomain: [0, mockValues.oneD.length],
+        showGrid: true,
+      }}
+      ordinateConfig={{ visDomain: domain, showGrid: true }}
+    >
+      <Pan />
+      <Zoom />
+      <AxialSelectToZoom {...args} />
+      <ResetZoomButton />
+
+      <DataCurve
+        abscissas={getAxisValues(undefined, mockValues.oneD.length)}
+        ordinates={mockValues.oneD}
+        errors={mockValues.oneD_errors}
+        color="blue"
+        showErrors
+      />
```

Comment:
Adding a curve so the zoom behaviour is clearer, like the image in `SelectToZoom`.